packages/next/client/trusted-types.ts

```diff
@@ -0,0 +1,19 @@
+let policy: TrustedTypePolicy | undefined
+
+function getPolicy() {
+  if (!policy && typeof window !== 'undefined') {
+    policy = window.trustedTypes?.createPolicy('nextjs', {
```

Comment:
Make sure to ask in the upstream pull request if they are happy with `nextjs` as the policy name, since it won't be trivial to change once people start using Trusted Types with Next.js.

Reply:
Got it, I have now added this to the pull request description.